app/component/favourites/favourite-stop-card-list-container.cjsx

```diff
@@ -0,0 +1,30 @@
+React             = require 'react'
+Relay             = require 'react-relay'
+queries           = require '../../queries'
+StopCardContainer = require '../stop-cards/stop-card-container'
+StopCardList      = require '../stop-cards/stop-card-list'
+
+DEPARTURES_COUNT = 5
+
+class FavouriteStopCardListContainer extends React.Component
+
+  proptypes: {
+    stops: React.PropTypes.any.isRequired
+  }
+
+  getStopCards: =>
+    stopCards = []
+    for stop in @props.stops.stops
+      stopCards.push <StopCardContainer key={stop.gtfsId} stop={{stop:stop}} departures=DEPARTURES_COUNT />
```

Comment:
This also has unnecessary curly braces. Also, there should be a space added after the colon, probably would be a good idea to run coffeelint, with the config from the root folder.